packages/server/src/api/routes/tests/role.spec.ts

```diff
@@ -149,6 +159,19 @@ describe("/roles", () => {
         { status: 400, body: { message: LOOP_ERROR } }
       )
     })
+
+    it("handle updating a role, without its inherits", async () => {
+      const res = await config.api.roles.save({
+        ...basicRole(),
+        inherits: [BUILTIN_ROLE_IDS.ADMIN],
+      })
+      // save again
+      const updatedRes = await config.api.roles.save({
+        ...res,
+        inherits: undefined,
+      })
+      expect(updatedRes.inherits).toEqual([BUILTIN_ROLE_IDS.ADMIN])
```

Comment:
I'm not sure I understand why we expect this behaviour. I don't think many other things in our API work this way. How come you can't erase/reset the `inherits` field to `undefined` or some default value?

Reply:
Discussed in Adri's comment, this is old behaviour being tested for now.